src/puppetlabs/trapperkeeper/services/webrouting/webrouting_service_core.clj

```diff
@@ -24,6 +24,9 @@
 (def ContextHandlerOptions
   (dissoc (merge jetty9-core/ContextHandlerOptions RouteOption) :server-id))
 
+(def WebsocketHandlerOptions
+  (dissoc (merge jetty9-core/CommonOptions RouteOption) :server-id))
```

Comment:
Just a sanity check: so, so far, we don't have any new options that are specific to a websocket handler?

It looks like we don't have a schema for `RingHandlerOptions`, so I'm wondering if we need a schema for this one?  At least until we have some new options that are specific to it?  (it looks like `add-ring-handler` just uses `RouteOption` directly?)

I don't have a strong opinion about this, just taking a survey of the existing code.  @fpringvaldsen do you have any thoughts?